### PR TITLE
Add frontend OWNERS to demo plugin

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: dynamic-demo-plugin/OWNERS [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - jeff-phillips-18
+  - jhadvig
+  - kyoto
+  - rhamilto
+  - spadgett
+  - TheRealJon
+  - zherman0
+approvers:
+  - andrewballantyne
+  - bparees
```
Comment (spadgett) [inline dynamic-demo-plugin/OWNERS:13]: I'd remove @bparees here

File: dynamic-demo-plugin/OWNERS [hunk 2]
```diff
@@ -0,0 +1,23 @@
+reviewers:
+  - dtaylor113
+  - jcaianirh
+  - jeff-phillips-18
+  - jhadvig
+  - kyoto
+  - rhamilto
+  - spadgett
+  - TheRealJon
+  - zherman0
```
Comment (spadgett) [inline dynamic-demo-plugin/OWNERS:10]: Maybe @vojtechszocs @rawagner @spadgett @christianvogt @andrewballantyne @glekner for reviewers.

File: dynamic-demo-plugin/OWNERS (hunk 1 above)
Comment (spadgett) [reply dynamic-demo-plugin/OWNERS:13]: @bparees I assume you don't want to be pinged for approvals :)

Comment (rawagner) [reply dynamic-demo-plugin/OWNERS]: OK, removed

Comment (bparees) [reply dynamic-demo-plugin/OWNERS]: fine by me :)

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/label px-approved
/label qe-approved
/label docs-approved

OWNERS file change only

**Pull request merged**